src/utils.mjs

```diff
@@ -71,6 +72,21 @@ export function isValidCheckpoint(checkpoint) {
   }
   return knowncheckpoints.indexOf(checkpoint) > -1;
 }
+
+export function isValidSourceTarget(checkpoint, source, target) {
+  switch (checkpoint) {
```

Comment:
We've been using maps of functions instead of `switch` so far.

This needs a test case, so that we can compare old audience values (not coming through) to new audience values (coming through)

Reply:
The checkpoint actually changed from `audiences` (old) to `audience` (new), so they should not get through anyways